packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -80,6 +79,7 @@ function compile<T>(regl: any, cmd: RawCommand<T>): CompiledReglCommand<T> {
 // draw calls, hitmap calls, and raycasting.
 
 export class WorldviewContext {
+  _lastPaintCall: number;
```

Comment:
nit: 
```suggestion
  _lastPaintCallTime: number = performace.now();
```

Reply:
That could throttle the first call to paint, I think. Best to leave it off, I think